### PR TITLE
slider组件

### PR DESCRIPTION
**Pull request by zhang0821**
slider组件

### CODE REVIEW

**Comment by zhang0821** [comment]
slider

File: src/fields/fieldSlider.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            }
+            return '';
+        },
+    },
+    methods: {
+        handleChange(value) {
+            this.$set(this.form.model, this.field.model, value);
+            this.$emit('on-change', this.field.model, value, null, this.field);
+        },
+        handleInput(value) {
```
Comment (zhengxkq) [inline src/fields/fieldSlider.vue]: I think this method not use ?

File: src/formGenerator.vue [hunk 2]
```diff
@@ -333,7 +334,17 @@ export default {
             this.$refs.form.validateField(model);
             this.$emit('on-field-change', model, value);
         },
-
+        handleFieldInput({model, value}) {
```
Comment (zhengxkq) [inline src/formGenerator.vue]: he same as above

**Review by zhengxkq — CHANGES_REQUESTED** [review]
I found all of handleInput method don't use, I think you can delete these method, thanks!

**Review by zhengxkq — APPROVED** [review]

**Pull request merged**